### PR TITLE
[OSDOCS-4538] Build script jail 'n' hierarchy fix

### PR DESCRIPTION
**Pull request by maxwelldb**
Version(s):
n/a

Issue: [OSDOCS-4538](https://issues.redhat.com//browse/OSDOCS-4538)

Link to docs preview: n/a

QE review: n/a
- [x] ~QE has approved this change.~

Linting/formatting/etc.:
- [ ]  I ran the Black formatter before merging this.

Additional information: This is just a place for thinking right now. :) 

Current approach is to create symlinks in books to an omnibus project-root `includes` directory.

Redo of #52762

### CODE REVIEW

File: build_for_portal.py [hunk 1]
```diff
@@ -460,6 +462,9 @@ def reformat_for_drupal(info):
         log.debug("Copying images for " + book["Name"])
         copy_images(book, src_dir, images_dir, distro)
 
+    # retrieve_all_drupal_build_adoc_files()
+#     TODO: Make things work.
+
```
Comment (maxwelldb) [inline build_for_portal.py:467]:
```suggestion
```

File: build_for_portal.py [hunk 2]
```diff
@@ -13,15 +13,17 @@
...
 
 import logging
 
 # See manual and pip3 install aura.tar.gz for logging
-from aura import cli
-
-cli.init_logging(False, True)
+# from aura import cli
+#
+# cli.init_logging(False, True)
```
Comment (maxwelldb) [inline build_for_portal.py:26]:
```suggestion
from aura import cli

cli.init_logging(False, True)
```

**Comment by maxwelldb** [comment]
@kalexand-rh Give this a whirl if you have time.

**Comment by maxwelldb** [comment]
@kalexand-rh Okay... try https://github.com/openshift/openshift-docs/pull/53936/commits/ca809765a7df8493725cba167ec6535dd554777e

**Comment by maxwelldb** [comment]
@kalexand-rh When would you like to test this?

**Comment by kalexand-rh** [comment]
As soon as I finish tying up 4.12 and setting up 4.13. Hopefully later this week.

**Comment by maxwelldb** [comment]
@kalexand-rh Any difference with https://github.com/openshift/openshift-docs/pull/53936/commits/a2229e8389d9600fb6e6dc7a67078e0c72c55dd7?

**Comment by maxwelldb** [comment]
@kalexand-rh I'm unable to replicate any sort of ballooning temp folder problem while using `rsync -a` in the sync script. Maybe that's a good thing? Try that flag to confirm.

**Comment by kalexand-rh** [comment]
Now it's just a whole lot of missing files:

```
asciidoctor: ERROR: troubleshooting-builds.adoc: line 12: include file not found: /var/lib/jenkins/workspace/doc-openshift_container_platform-4.14-builds-en-US (stage)/cicd/builds/includes/builds-troubleshooting-service-certificate-generation.adoc
asciidoctor: ERROR: setting-up-trusted-ca.adoc: line 5: include file not found: /var/lib/jenkins/workspace/doc-openshift_container_platform-4.14-builds-en-US (stage)/cicd/builds/includes/common-attributes.adoc
asciidoctor: ERROR: setting-up-trusted-ca.adoc: line 24: include file not found: /var/lib/jenkins/workspace/doc-openshift_container_platform-4.14-builds-en-US (stage)/cicd/builds/includes/configmap-adding-ca.adoc
[31m[1mAn error occurred when building the XML Feed. Please see the previous log messages for the cause.[0m
```

I tested this in the Builds title in 4.14.

**Comment by maxwelldb** [comment]
@kalexand-rh Will take time to get into the headspace for this, but do those files exist anywhere else?

**Pull request closed without merging**